src/map/battle.cpp

```diff
@@ -5925,12 +5925,12 @@ struct Damage battle_calc_magic_attack(struct block_list *src,struct block_list
 						if(sd && sd->spiritcharm_type == CHARM_TYPE_WATER && sd->spiritcharm > 0)
 							skillratio += 25 * sd->spiritcharm;
 						break;
-					case NJ_RAIGEKISAI:
+					case NJ_RAIGEKISAI: // !TODO: Confirm new formula
 						skillratio += 60 + 40 * skill_lv;
 						if(sd && sd->spiritcharm_type == CHARM_TYPE_WIND && sd->spiritcharm > 0)
 							skillratio += 15 * sd->spiritcharm;
 						break;
-					case NJ_KAMAITACHI:
+					case NJ_KAMAITACHI: // !TODO: Confirm new formula
```

Comment:
New formulas are 
NJ_HUUMA: skillratio += -150 + 250 * skill_lv;
NJ_KASUMIKIRI: skillratio += 20 * skill_lv;
NJ_KIRIKAGE: skillratio += -50 + 150 * skill_lv;
NJ_KUNAI: skillratio += -100 + 100 * skill_lv;
NJ_RAIGEKISAI (not sure if you want to apply the spirit charm update with this PR though): 
```
skillratio += 100 * skill_lv;
if(sd && sd->spiritcharm_type == CHARM_TYPE_WIND && sd->spiritcharm > 0)
	skillratio += 20 * sd->spiritcharm;
break;
```
NJ_KAMAITACHI: no change